applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -84,6 +84,7 @@ static void advertising_process(struct k_work *work)
 {
 	int ret;
 	struct bt_le_adv_param adv_param;
+	struct bt_le_ext_adv *adv;
 
 #if CONFIG_BT_BONDABLE
```

Comment:
A check: If we set BONDABLE then both standard and extended adv will be used?

Reply:
If we set BONDABLE, the headset will use ADV-EXT if the bonding info cannot be found.
Once if headset is bonded, then it will use direct ADV.